proto/decentraland/kernel/apis/communications_controller.proto

```diff
@@ -11,8 +11,14 @@ message SendBinaryRequest {
   repeated bytes data = 1;
 }
 
+message SendBinaryToAddress {
+  bytes data = 1;
+  optional string address = 2;
+}
+
 message SendBinaryResponse {
   repeated bytes data = 1;
+  repeated SendBinaryToAddress messages_to_address = 2;
```

Comment:
I dont like this name, maybe: 
- `targeted_messages`
- `private_messages`
- `targeted_messages`
- `address_messages`